mpf/devices/ball_hold.py

```diff
@@ -10,7 +10,7 @@
 from mpf.core.system_wide_device import SystemWideDevice
 
 
-@DeviceMonitor("balls_held", enabled="_enabled")
+@DeviceMonitor("balls_held", "enabled")
```

Comment:
This will fix evaluations for one shot placeholders. However, it will not allow subsections because the placeholder will not notice changes to the getter. That is the reason why I tried to use the internal variable. Maybe I just inverted the order? It is good that we have tests now. Maybe we can create a generic test which just queries all placeholders to make sure that they do not crash at least?

Reply:
Hey @jabdoa2 , thanks for the feedback. I tried swapping the positions to `_enabled="enabled" and my new tests still pass, is that sufficient? Could you provide tests for the situation you're describing above?

The trouble with the current bug is that it doesn't actually crash, the placeholder substitution swallows the error and returns None. But I'm all in favor of more tests!

Reply:
This can be tested like this: https://github.com/missionpinball/mpf/commit/1239ef4c9feb27c09b39ccd53a711976e7280aed#diff-e74bb5ba8adfd77096ec130b2124abf4R16. I'm pretty confident that your solution is right!

Reply:
I've pushed an update that restores the kwargs but swaps the underscore position. All the tests (including my new ones) pass, so... hurray?